would you mind sharing how did you compile it? I'm facing errors when executing `make deps`; I'm using docker and mounting the project's folder:

```bash
$ docker run -it --rm --entrypoint=/bin/bash -v ${PWD}:/data -w /data golang:1.22

root@aecc0a0026ee:/data# make deps
GO111MODULE=on go mod vendor
go: downloading github.com/hashicorp/go-multierror v1.0.0
go: downloading github.com/urfave/cli v1.22.4
go: downloading k8s.io/api v0.17.0
go: downloading k8s.io/apimachinery v0.17.5
go: downloading gopkg.in/yaml.v2 v2.4.0
go: downloading github.com/stretchr/testify v1.4.0
go: downloading github.com/namely/k8s-configurator v0.0.4
go: downloading github.com/pkg/errors v0.8.1
go: downloading k8s.io/client-go v11.0.0+incompatible
go: downloading github.com/hashicorp/errwrap v1.0.0
go: downloading github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d
go: downloading github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
go: downloading k8s.io/klog v1.0.0
go: downloading github.com/davecgh/go-spew v1.1.1
go: downloading github.com/pmezard/go-difflib v1.0.0
go: downloading github.com/ghodss/yaml v0.0.0-20150909031657-73d445a93680
go: downloading gopkg.in/inf.v0 v0.9.1
go: downloading github.com/google/gofuzz v1.0.0
go: downloading sigs.k8s.io/yaml v1.1.0
go: downloading github.com/russross/blackfriday/v2 v2.0.1
go: downloading github.com/shurcooL/sanitized_anchor_name v1.0.0
go: downloading golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4
go: downloading github.com/json-iterator/go v1.1.8
go: downloading github.com/modern-go/reflect2 v1.0.1
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading golang.org/x/text v0.3.3
go install github.com/mattn/goveralls
cannot find module providing package github.com/mattn/goveralls: import lookup disabled by -mod=readonly
	(Go version in go.mod is unspecified, so vendor directory was not used.)
make: *** [Makefile:14: deps] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -11,8 +11,8 @@ test:
 .PHONY: deps
 deps:
 	GO111MODULE=on go mod vendor
-	go install github.com/mattn/goveralls
-	go install github.com/go-playground/overalls
+	go install github.com/mattn/goveralls@0.12
+	go install github.com/go-playground/overalls@latest
 
 .PHONY: coveralls
 coveralls:

